dev/PushNotifications/PushRawNotification.h

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation and Contributors.
+// Licensed under the MIT License.
+
+#include <winrt/Windows.Networking.PushNotifications.h>
+#include <winrt/Windows.ApplicationModel.background.h>
+#include <winrt/Windows.Storage.Streams.h>
+#include <winrt/Windows.Security.Cryptography.h>
+
+struct PushRawNotification : winrt::implements<PushRawNotification, winrt::Windows::Networking::PushNotifications::IRawNotification, winrt::Windows::Networking::PushNotifications::IRawNotification2, winrt::Windows::Networking::PushNotifications::IRawNotification3>
```

Comment:
The goal of PushRawNotification was to mock the RawNotification object returned by BI. This allows us to have a single constructor that accepts an IBackgroundTaskInstance instead of just wrapping the string as IInspectable and needing two constructors to handle a string or RawNotification.

Reply:
Closing comment after discussing offline, no changes needed 😊.